exercises/D-strings/exercise.js

```diff
@@ -1,3 +1,4 @@
 // Start by creating a variable `message`
-
+let message = "This is a string";
```

Comment:
Brilliant work, this does exactly what is needed. The only comment would be to change `let` to `const` since we never change the value of `message`. Good job Shima.